.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,19 +1,16 @@
+---
+name: Bug report
+about: Create a bug report to help improve containerd
+title: ''
+labels: kind/bug
+assignees: ''
+---
+
 <!--
 If you are reporting a new issue, make sure that we do not have any duplicates
 already open. You can ensure this by searching the issue list for this
 repository. If there is a duplicate, please close your issue and add a comment
 to the existing issue instead.
-
-If you suspect your issue is a bug, please edit your issue description to
```

Comment:
why did you remove this part?

Reply:
I can add some variant of it back in, but there are no commands in the bug report information section to run, i.e. `ctr --version` so it doesn't seem very helpful. What do you suggest?

Reply:
ok, makes sense :)